### PR TITLE
Replication enhancements following NHA fixes

### PR DESCRIPTION
**Pull request by ross-spencer**
WIP following up on enhancements for NHA. 

PR also:

* Updates use of markdown for headings where previously our syntax limited our headings from `<h1>` to `<h5>` we can now access `<h6>` on the administration page. Headings reference: https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#headings. We use `<h6>` to create a more delineated structure under Replicators. 

* Makes note of Tessa's management command for creating replicas in bulk.

**Notes**

* This will need to be merged once a `0.18` docs branch has been created. The target for this PR can then be updated.

Connected to archivematica/Issues#685
Connected to https://github.com/archivematica/Issues/issues/985
Connected to https://github.com/archivematica/Issues/issues/359
Connected to https://github.com/archivematica/Issues/issues/1209

### CODE REVIEW

File: administrators.rst [hunk 1]
```diff
@@ -882,14 +882,83 @@ Replicator
...
+storage locations. Replicators can be configured for each of the following
+protocols supported by the storage service:
+
+* Local filesystem, including GPG encrypted.
+* S3.
+* Arkivum.
+* Duracloud.
+* DSpace via SWORD2 API.
+* DSpace via REST API.
+* Fedora via Sword2
```
Comment (evelynPM) [inline administrators.rst]: Add missing period

File: administrators.rst [hunk 2]
```diff
@@ -882,14 +882,83 @@ Replicator
...
+* LOCKSS-o-matic.
+* NFS.
+* Pipeline local filesystem.
+* Swift.
+
+Replicators are associated with an AIP storage location via the storage
+location's configuration page. As such Replicators must be configured before
+they can be selected for use alongside an AIP store. There is additional
+information about this under :ref:`configure location <configure_location>`. If
+you wish for the replicated AIPs to be encrypted, create the location in an
```
Comment (evelynPM) [inline administrators.rst]: Change "wish for the replicated AIPs" to "would like your replicated AIPs"

File: administrators.rst [hunk 3]
```diff
@@ -882,14 +882,83 @@ Replicator
...
 
+Outcomes of a Replicator
+""""""""""""""""""""""""
+
+**One AIP to many replicators:**
+
+An AIP will be replicated *n-* times where there are *n-* configured
+Replicators for an AIP storage location.
+
+**On reingest replicas are replaced and given new identifiers:**
```
Comment (evelynPM) [inline administrators.rst]: Add comma after "reingest"

File: administrators.rst [hunk 4]
```diff
@@ -882,14 +882,83 @@ Replicator
...
+
+During reingest, previous replicas will be deleted and marked as DELETED within
+the storage service Packages tab. A new replica will be created and marked as
+UPLOADED. The new replica will be assigned a new UUID. Pointer files will be
+updated with accurate references between the source and the replica and vice
+versa.
+
+**Reingest can be used to update or delete replicas:**
+
+If a Replicator is disabled then reingest can be used to selectively remove
```
Comment (evelynPM) [inline administrators.rst]: Change "If a Replicator is disabled then reingest can be used" to "If a Replicator is disabled, reingest can be used"

File: administrators.rst [hunk 5]
```diff
@@ -882,14 +882,83 @@ Replicator
...
+replicas from that replica location. If a new Replicator is enabled then
```
Comment (evelynPM) [inline administrators.rst]: Change "If a new replicator is enabled then" to "If a new replicator is enabled," Also, capitalize "replicator".

File: administrators.rst [hunk 6]
```diff
@@ -882,14 +882,83 @@ Replicator
 ^^^^^^^^^^
 
 Replicator locations can be configured to replicate the AIPs in one or more AIP
-storage locations. If you wish for the replicated AIPs to be encrypted, create
-the location in an :ref:`encrypted space <gpg>`.
+storage locations. Replicators can be configured for each of the following
+protocols supported by the storage service:
```
Comment (evelynPM) [inline administrators.rst]: Capitalize "storage service" here and elsewhere in docs?

File: administrators.rst [hunk 7]
```diff
@@ -882,14 +882,83 @@ Replicator
...
 
 .. note::
 
    If you want the same space to have multiple purposes, multiple locations with
    different purposes can be created.
 
+Outcomes of a Replicator
+""""""""""""""""""""""""
+
+**One AIP to many replicators:**
```
Comment (evelynPM) [inline administrators.rst]: Make capitalization consistent - "Replicator" is capitalized in most parts of this document

File: administrators.rst [hunk 8]
```diff
@@ -882,14 +882,83 @@ Replicator
...
+Limitations of a Replicator
+"""""""""""""""""""""""""""
+
+**Tight-coupling with AIP storage in a Pipeline:**
+
+Replication is coupled with AIP storage. As such, if there is a failure when
+storing a replica, even if the source AIP has succeeded, an Archivematica
+pipeline still will fail during processing. This affects indexing of the AIP in
+the Archival Storage tab within Archivematica. It also leaves the replica
+packages within STAGING in the storage service and will be marked as such.
```
Comment (evelynPM) [inline administrators.rst]: Not quite clear what "and will be marked as such" means

File: administrators.rst [hunk 9]
```diff
@@ -882,14 +882,83 @@ Replicator
...
+
+**New UUIDs during reingest:**
+
+Given a new UUID during reingest, external services tracking the packages in
```
Comment (evelynPM) [inline administrators.rst]: Change to "Since the AIP will be given a new UUID during reingest"

File: administrators.rst [hunk 10]
```diff
@@ -911,7 +980,19 @@ To create and configure a new Location:
       create it manually and make sure it is writable by the Archivematica
       user.
 
-#. For an AIP storage location, choose the replicator location(s) if desired.
+#. For an AIP storage location, choose replicator location(s) if desired. To
+   select more than one hold down ctrl- while clicking the UUID of the replica.
```
Comment (evelynPM) [inline administrators.rst]: Add comma after "one"

File: administrators.rst [hunk 11]
```diff
@@ -911,7 +980,19 @@ To create and configure a new Location:
...
+
+   .. note::
+
+      One or more replicator locations must already have been configured to be
+      selected. If a replicator location is configured afterwards then it can
+      be added via this page.
```
Comment (evelynPM) [inline administrators.rst:991]: Which page?

File: administrators.rst [hunk 12]
```diff
@@ -911,7 +980,19 @@ To create and configure a new Location:
...
+
+      Replica AIPs will only be created after a location has been associated
```
Comment (evelynPM) [inline administrators.rst]: Change "will only be created after" to "will be created only after"

File: administrators.rst [hunk 13]
```diff
@@ -911,7 +980,19 @@ To create and configure a new Location:
...
+      with a Replicator but can also be created retroactively via the storage
```
Comment (evelynPM) [inline administrators.rst]: Add comma after Replicator

**Review by sarah-mason — APPROVED** [review]
I think if you address Evelyn's comments that should fix everything -- so I have no other comments to add other than nice work!

**Comment by ross-spencer** [comment]
thanks @evelynPM and @sarah-mason changes made and merged. This PR is not showing as merged at time of writing - I think GH is having sync issues, but should be okay sometime later today. It's very much in the `0.18` branch now.

**Pull request closed without merging**